api/src/main/java/org/openmrs/CohortMember.java

```diff
@@ -34,13 +34,18 @@
     public CohortMember() {
     }
 
-	public CohortMember(Integer cohortMemberId, Integer cohortId, Integer patientId, Date startDate, Date endDate) {
+	public CohortMember(Integer cohortMemberId, Cohort cohortId, Patient patientId, Date startDate) {
 		this();
 	    this.cohortMemberId = cohortMemberId;
 		this.cohortId = cohortId;
 		this.patientId = patientId;
 		this.startDate = startDate;
-		this.endDate = endDate;
+	}
+
```

Comment:
These are the constructors I currently have in CohortMember.java, while I discarded the rest.

```
	public CohortMember(Patient patientId, Date startDate) {
		this.patientId = patientId;
		this.startDate = startDate;
	}

	public CohortMember(Patient patientId) {
		this(patientId, new Date());
	}

```